caffe2/operators/layer_norm_op.cc

```diff
@@ -182,99 +182,18 @@ to the end.)
     .Output(1, "mean", "Mean values for each feature vector")
     .Output(2, "stddev", "Standard deviations for each feature vector");
 
-C10_DEFINE_OP_SCHEMA(LayerNorm, FunctionSchema(
-    "caffe2::layer_norm_dont_use_this_op_yet",
-    (std::vector<c10::Argument>{
-      c10::Argument("input"),
-      c10::Argument("axis", IntType::get()),
-      c10::Argument("epsilon", FloatType::get()),
-      c10::Argument("output", OptionalType::ofTensor(), nullopt, IValue()),
-      c10::Argument("output_mean", OptionalType::ofTensor(), nullopt, IValue()),
-      c10::Argument("output_stdev", OptionalType::ofTensor(), nullopt, IValue())
-    }), (std::vector<c10::Argument>{
-      c10::Argument("output"),
-      c10::Argument("mean"),
-      c10::Argument("stdev")
-    })
-));
-
 } // namespace caffe2
 
-
-// Register layer norm with c10
-namespace {
-struct Cache final : public c10::KernelCache {
-    at::optional<at::Tensor> scale = at::nullopt;
-    at::optional<at::Tensor> bias = at::nullopt;
-};
-
-template <class DataType>
-void layer_norm_c10(c10::Stack* stack, c10::KernelCache* cache_) { // TODO Pass in correct cache type
-  c10::ArrayRef<c10::IValue> inputs = torch::jit::peekSlice(*stack, 0, 3, 6);
-  c10::ArrayRef<c10::IValue> outputs = torch::jit::peekSlice(*stack, 3, 3, 6);
-
-
-  caffe2::Tensor X{c10::C10Tensor(inputs[0].toTensor())};
-  int64_t axis = inputs[1].toInt();
-  float epsilon = inputs[2].toDouble();
-
-  auto device = X.GetDevice();
-
-  caffe2::Tensor Y, mean, sig;
```

Comment:
ah, looks like you've deleted all that previous code from another diff - good :) (though makes it hard to review)